src/types/modifications/CO2eCalc.jl

```diff
@@ -53,7 +52,11 @@ function calc_co2e!(gen, ch4_gwp, ng_upstream_ch4_leakage, coal_upstream_ch4_lea
         if r.genfuel == "ng"
             r.emis_co2e = Container(r.emis_co2 .+ ng_upstream_ch4_leakage .* r.heat_rate .* ch4_gwp)
         elseif r.genfuel == "coal"
-            r.emis_co2e = Container(r.emis_co2 .+ coal_upstream_ch4_leakage .* r.heat_rate .* ch4_gwp)
+            if r.gentype == "coal_ccus_retrofit" && !hasproperty(gen, :capt_co2) # the capt_co2_percent has not been applied to emis_co2 yet if capt_co2 isn't in the gen table 
```

Comment:
@sallyrobson would it be better to change the ordering of the mods instead?  I'm not sure if there is anything forcing this CO2eCalc mod being before CCUS?

Reply:
honestly, I got confused by the ordering because from what I can tell the CCUS should be running before the CO2eCalc mod but for some reason it hasn't applied this change yet. I decided to do this fix for the sake of time but we could dig into it more in the future